app/src/main/java/com/amaze/filemanager/asynchronous/services/ftp/FtpService.java

```diff
@@ -324,7 +321,7 @@ public static InetAddress getLocalInetAddress(Context context) {
                 while (addresses.hasMoreElements()) {
                     InetAddress address = addresses.nextElement();
 
-                    if(WIFI_AP_ADDRESS.equals(address.getHostAddress()) && isEnabledWifiHotspot(context))
+                    if(address.getHostAddress().startsWith(WIFI_AP_ADDRESS_PREFIX) && isEnabledWifiHotspot(context))
```

Comment:
Please add a null check for address

Reply:
Please also add null check on FtpNotification#92 where the original crash happens.